pkg/env/dev.go

```diff
@@ -288,3 +288,7 @@ func (d *dev) E2EStorageAccountRGName() string {
 func (d *dev) E2EStorageAccountSubID() string {
 	return "0cc1cafa-578f-4fa5-8d6b-ddfd8d82e6ea"
 }
+
+func (d *dev) Type() EnvironmentType {
+	return EnvironmentTypeDevelopment
```

Comment:
I think `envType` should be a field in `type dev struct` with the value `EnvironmentTypeDevelopment` to be consistent with `type prod struct`.

```suggestion
func (d *dev) Type() EnvironmentType {
	return d.envType
```